webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -164,4 +198,14 @@ export class TasksComponent implements OnInit, OnDestroy {
   listTrackBy(index, task) {
     return task?._id;
   }
+
+  async getCurrentLineageLevel(){
+    return this.userContactService.get().then(user => user?.parent?.name);
+  }
+
+  async getLineagesFromTaskDocs(taskDocs){
+    const ids = _map(taskDocs, 'forId');
```

Comment:
Can you use native javascript `map` function?
`taskDocs.map(task => task.forId)`